dali/python/nvidia/dali/external_source.py

```diff
@@ -420,23 +420,37 @@ class ExternalSource():
 
     Acceptable sources depend on the value specified for ``batch`` parameter.
 
-    If batch is set to False, source must be a callable (a function or an object with __call__ method)
-    that accepts exactly one argument (:meth:`~nvidia.dali.types.SampleInfo` instance that
-    represents the index of the requested sample).
-    If batch is set to True, the ``source`` can be either a callable, an iterable or a generator function.
-    Callable in batch mode must accept exactly one argument - either :meth:`~nvidia.dali.types.BatchInfo`
-    instance or an integer (see `batch_info`).
+    If ``batch`` is set to ``False``, the ``source`` must be:
+
+      * a callable (a function or an object with __call__ method) that accepts exactly one argument
+        (:meth:`~nvidia.dali.types.SampleInfo` instance that represents the index of the requested sample).
+
+    If ``batch`` is set to ``True``, the ``source`` can be either:
```

Comment:
Add on more blank line to separate from the previous part.

Reply:
It breaks the documentation, I added it with `|`